commands/serve.js

```diff
@@ -20,6 +20,7 @@ module.exports = new Command('serve')
   .option('-o, --host <host>', 'the host on which to listen (default: localhost)', 'localhost')
   .option('--only <targets>', 'only serve specified targets')
   .option('--except <targets>', 'serve all except specified targets')
+  .option('--hosting-public <targets>', 'override public directory')
```

Comment:
I think "hosting-dir" might be a better flag name. Also should replace "\<targets\>" with "\<directory\>". @mbleigh what do you think?